freqtrade/plot/plotting.py

```diff
@@ -474,6 +475,7 @@ def load_and_plot_trades(config: Dict[str, Any]):
         pair_counter += 1
         logger.info("analyse pair %s", pair)
 
+        strategy.dp = DataProvider(config, config["exchange"])
```

Comment:
the 2nd argument to DataProvider should be an exchange object, not a part of the exchange configuration (a dictionary).

I wonder why mypy ain't screaming out loud, but i guess because the return of dict[] is "Any" - it might as well be a Exchange object.